hive-metastore/src/test/java/org/apache/iceberg/hive/HiveMetastoreTest.java

```diff
@@ -41,10 +44,21 @@ public abstract class HiveMetastoreTest {
 
   @BeforeClass
   public static void startMetastore() throws Exception {
+    startMetastore(Collections.emptyMap());
+  }
+
+  public static void startMetastore(Map<String, String> hiveConfOverride) throws Exception {
     HiveMetastoreTest.metastore = new TestHiveMetastore();
-    metastore.start();
+    HiveConf hiveConfWithOverrides = new HiveConf(new Configuration(), TestHiveMetastore.class);
```

Comment:
Is there a special reason behind not using `new HiveConf(TestHiveMetastore.class)` here?

Reply:
Not really, I just moved this line from `TestHiveMetastore`. Updated to remove this though.